std/range/package.d

```diff
@@ -1716,7 +1716,7 @@ pure @safe nothrow @nogc unittest
     }
 }
 
-/// https://issues.dlang.org/show_bug.cgi?id=24243
+/// $(LINK2 https://issues.dlang.org/show_bug.cgi?id=24243, Bug 24243)
 pure @safe nothrow unittest
```

Comment:
The correct fix here is to simply remove the extra slash so that it's not a ddoc comment any longer. Tests like this have the link to the bug on them as a comment so that maintainers know what they're testing for. They're not supposed to be part of the documentation.

Reply:
Done